src/Molecules/Filter/Filter.jsx

```diff
@@ -1,8 +1,10 @@
 import React, { useState } from 'react'

```

Comment:
Would this be better if it was moved in `TableList` ? Because we now have the icon + popover in this. Dunno tho

Reply:
Yeah, later, let's close all theses stories